release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -667,6 +667,146 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-10-monitoring-stack-components-and-dependencies"]
+==== Monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.23.0
+* Grafana to 8.3.4
+* kube-state-metrics to v2.3.0
+* node-exporter to 1.3.1
+* prom-label-proxy to 0.4.0
+* Prometheus to 2.32.1
+* Prometheus adapter to 0.9.1
+* Prometheus operator to 0.53.1
+* Thanos to 0.23.1
+
+[id="ocp-4-10-monitoring-new-metrics-targets-page-in-web-console"]
+==== New page for metrics targets in web console
+
+A new *Metrics Targets* page in the {product-title} web console shows targets for default {product-title} projects and for user-defined projects. You can use this page to view, search, and filter the endpoints that are currently targeted for scraping, which helps you to identify and troubleshoot problems.
+
+[id="ocp-4-10-monitoring-components-use-TLS-authentication-for-metrics"]
+==== Monitoring components updated to use TLS authentication for metrics collection
+
+With this release, all monitoring components are now configured to use mutual TLS authentication rather than Bearer Token static authentication for metrics collection. This authentication method is more resilient when an outage occurs in the Kubernetes API. It also decreases the load on the Kubernetes API.
```

Comment:
```suggestion
With this release, all monitoring components are now configured to use mutual TLS authentication, rather than Bearer Token static authentication, for metrics collection. TLS is more resilient to Kubernetes API outages, and decreases the load on the Kubernetes API.
```